src/components/Carousel.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable */
+alert('Redundant alt attribute. Screen-readers already announce img tags as an image. You don’t need to use the words image, photo, or picture (or any specified custom words) in the alt prop jsx-a11y/img-redundant-alt');
```

Comment:
I don't think this is the right syntax (if you notice, it executes the alert) - you should instead do something like

```js
/* eslint-disable jsx-a11y/img-redundant-alt */
```

Reply:
Oh okay, It didnt give me an error so I thought I was good XD. Thanks for that!